src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/SimpleConfigModel.cs

```diff
@@ -571,6 +577,14 @@ private IGlobalRunConfig ProduceOperationSetup(SpecialOperationConfigParams defa
                             }
                         }
                     }
+
+                    if (symbol.Value is DerivedSymbol derivedSymbol)
```

Comment:
This one should move to above the replaces check (to set up the map between the value of the source symbol and the value of the derived symbol), then the block for handling the value forms of parameter symbols should be made to handle BaseSomethingSymbol